lib/models/organization_member_model.dart

```diff
@@ -36,15 +36,15 @@ class OrganizationMemberModel {
   final String? pronouns;
   final String? role;
   final String? socials;
-  final String? status;
+  final List? status;
```

Comment:
You did not mention that you worked on this file?

Reply:
I did